contentcuration/contentcuration/models.py

```diff
@@ -377,29 +378,43 @@ class ContentNode(MPTTModel, models.Model):
     objects = TreeManager()
 
     def get_original_node(self):
-
+        key = "original_channel_{}".format(self.original_source_node_id or self.pk)
```

Comment:
If we cache this, won't we then still have the problem of the original channel's chef being re-run and the cached target node no longer being in there?